pages/services/babydegen.jsx

```diff
@@ -103,38 +89,166 @@ const WhatIsBabyDegen = () => (
   </SectionWrapper>
 );
 
-const MeetTheOptimusAgent = () => (
-  <SectionWrapper customClasses="lg:p-24 border-y border-[#FFE0E6] w-full h-full bg-gradient-to-r from-[#FFF0F1] to-white to-70%">
-    <div className="px-8 py-12 lg:p-0 grid max-w-screen-xl lg:px-16 mx-auto lg:gap-8 xl:gap-0 lg:grid-cols-12 items-center">
-      <div className="lg:col-span-6 text-center px-5 lg:p-0 lg:text-left lg:w-6/7">
-        <h1 className={`${SUB_HEADER_CLASS} mb-8 mt-12`}>
-          Meet the Optimus agent: a step towards BabyDegen
-        </h1>
-        <p className="mb-8">
-          The Optimus Agent is an autonomous AI agent that streamlines your DeFi
-          experience by intelligently managing your assets on specific
-          blockchain platforms. Initially focused on select DeFi protocols on
-          Optimism Mainnet and Base, it offers a targeted approach to maximizing
-          returns within these ecosystems.
-        </p>
-        <p className="mb-8">
-          The Optimus Agent is currently in alpha testing via Quickstart and
-          currently supports key protocols like Balancer and Uniswap. It’s
-          operational on Ethereum, Base, and Optimism. Looking ahead, it will
-          expand to support any protocol that wishes to integrate.
-        </p>
+const taggedItem = (item) => (
+  <div className="max-sm:mt-2 rounded-full border py-1 pr-2 max-w-fit inline mr-2">
+    <Image
+      src={`/images/services/babydegen/${item}.png`}
+      alt={item}
+      width={30}
+      height={30}
+      className="inline p-1 pb-2"
+    />
+    {item}
+  </div>
+);
+
+const modiusKeyFeatures = [
+  'Personalized portfolio management.',
+  'Expanding strategy library.',
+  <>Blockchain network: {taggedItem('Mode')}</>,
```

Comment:
this doesn't seem to work probably because the icons are in low case 
<img width="622" alt="image" src="https://github.com/user-attachments/assets/9b864681-f47d-4ccd-87b1-2815f1e90584" />